src/test/java/uk/gov/companieshouse/repository/dissolution/DissolutionRepositoryTest.java

```diff
@@ -39,8 +39,8 @@ public void findByCompanyNumber_findsActiveDissolution() {
     }
 
     @Test
-    public void findByCompanyNumber_DoesNotFindInactiveDissolution() {
-        final String COMPANY_NUMBER = "123";
+    void findByCompanyNumber_DoesNotFindInactiveDissolution() {
+        final String COMPANY_NUMBER = "225"; //reference should be unique everytime you test
```

Comment:
Yea this is just a random number. Just a reminder before running the test to change it because it took us a long time to figure out why it was failing

Reply:
It might help to add to the comment that this is a random number.